KarhooSDK/Api/Response/Adyen/AdyenPaymentMethod.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  AdyenPaymentMethod.swift
+//  KarhooSDK
+//
+//  Copyright © 2020 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+
+public struct AdyenPaymentMethod: KarhooCodableModel {
+    public let brands: [String]
+    public let configuration: [String: String]
+    public let details: [InputDetail]
+    public let group: [Group]
+    public let items: [Item]
+    public let paymentMethodData: String
+    public let supportsRecurring: Bool
+    public let type: String
+    
+    public init(brands: [String] = [],
+                configuration: [String: String] = [String: String](),
+                details: [InputDetail] = [],
+                group: [Group] = [],
+                items: [Item] = [],
+                paymentMethodData: String = "",
+                supportsRecurring: Bool,
+                type: String) {
+        self.configuration = configuration
+        self.brands = brands
+        self.details = details
+        self.group = group
+        self.items = items
+        self.paymentMethodData = paymentMethodData
+        self.supportsRecurring = supportsRecurring
+        self.type = type
+    }
+    
+    enum CodingKeys: String, CodingKey {
+        case brands
+        case configuration
+        case details
+        case group
+        case items
+        case paymentMethodData
+        case supportsRecurring
```

Comment:
these should match the API keys in the response - eg

```
case paymentMethodData = "payment_method_data"
``` 
unless the json is returning paymentMethodData in camel case its fine :-)

Reply:
If the BE is matching the Adyen response, then it will be in camel case. Currently there's no documentation on this although we've been told it's just a wrapper so I'm assuming it'll match.